openverse_catalog/dags/providers/provider_dag_factory.py

```diff
@@ -92,7 +92,7 @@
 _DATE_RANGE_INNER_TEMPLATE = "macros.ds_add(ds, -{} )"
 DATE_RANGE_ARG_TEMPLATE = "{{{{" + _DATE_RANGE_INNER_TEMPLATE + "}}}}"
 DATE_PARTITION_ARG_TEMPLATE = Template(
-    "$media_type/$provider_name/{{ date_partition_for_prefix(dag.schedule, dag_run.logical_date, $reingestion_date ) }}"  # noqa
+    "$media_type/$provider_name/{{ date_partition_for_prefix(dag.schedule_interval, dag_run.logical_date, $reingestion_date ) }}"  # noqa
```

Comment:
This is so odd! I thought maybe `dag.params['schedule']` might work, but no:

```ERROR - Failed to execute task: 'airflow.models.param.ParamsDict object' has no attribute 'schedule'.```

Reply:
It definitely made me do a double take at first 😞